test/functions.php

```diff
@@ -16,28 +16,24 @@ public function testErrorDBCheckEmptyInput() {
     }
 
     public function testErrorDBCheckStringInput() {
-        $expected = 'Unexpected error. Please refresh page.';
+        $this->expectException(TypeError::class);
         $actual = DBCheck('test');
-        $this->assertEquals($expected,$actual);
     }
 
     public function testErrorDBCheckBoolInput() {
-        $expected = 'Unexpected error. Please refresh page.';
+        $this->expectException(TypeError::class);
         $actual = DBCheck(true);
-        $this->assertEquals($expected,$actual);
-    }
+        }
 
     public function testErrorDBCheckIntInput() {
-        $expected = 'Unexpected error. Please refresh page.';
+        $this->expectException(TypeError::class);
         $actual = DBCheck(1);
-        $this->assertEquals($expected,$actual);
-    }
+        }
 
     public function testErrorDBCheckFloatInput() {
-        $expected = 'Unexpected error. Please refresh page.';
+        $this->expectException(TypeError::class);
         $actual = DBCheck(1.1);
```

Comment:
dunno if i'm dumb but would you be able to change DBCheck(1) and DBCheck(1.1) to DBCheck(0) and DBCheck(1)? don't know if i'm mistaking this for something else :)